PyRDF/backend/Local.py

```diff
@@ -13,6 +13,41 @@ class Local(Backend):
         backend.
 
     """
+    def __init__(self, config={}):
+        """
+        Creates a new instance of the
+        Local implementation of `Backend`.
+
+        Parameters
+        ----------
+        config (optional)
+            The config object for the required
+            backend. The default value is an
+            empty Python dictionary `{}`.
+
+        """
+        super().__init__(config)
+        self.supported_operations = [
+        'Define',
+        'Filter',
+        'Histo1D',
+        'Histo2D',
+        'Histo3D',
+        'Profile1D',
+        'Profile2D',
+        'Profile3D',
+        'Count',
+        'Min',
+        'Max',
+        'Mean',
+        'Sum',
+        'Fill',
+        'Report',
+        'Range',
```

Comment:
We can leave range in the list only if the local backend is not configured to use multi-threading.

Reply:
Ping :D

Reply:
@etejedor , done now :+1: